source/common/grpc/common.h

```diff
@@ -35,6 +35,13 @@ class Common {
   static Optional<Status::GrpcStatus> getGrpcStatus(const Http::HeaderMap& headers);
 
   /**
+   * Returns the grpc-message from a given set of headers, if present.
+   * @param headers the headers to parse.
+   * @return the gRPC message string or empty string if grpc-message is not present in headrs.
```

Comment:
`@return std::string the gRPC message ...` (types first in `@return` doc comments)